src/test/java/pages/AccountsPage.java

```diff
@@ -0,0 +1,46 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class AccountsPage extends BasePage {
+    private final static By NEW_ACCOUNT_BUTTON = By.cssSelector("div.active a[title=New]");
+    private final static By ICON_ACCOUNTS_LOCATOR = By.xpath("//records-highlights-icon");
+    By actualAccountNameLocator = By.xpath("//h1/div[contains(text(),'Account')]/parent::h1/descendant::lightning-formatted-text");
```

Comment:
Please create a separate page class for these elements: AccountDetailsPage and the same for Leads.

Reply:
AccountsPage:
<img width="1679" alt="image" src="https://user-images.githubusercontent.com/14195681/181803742-0d8887aa-c7dc-4daa-af26-9b14879fa7ab.png">

AccountDetailsPage:
<img width="1666" alt="image" src="https://user-images.githubusercontent.com/14195681/181803789-5988e316-a33e-4ba8-b189-8dbce3312310.png">